docs/engineering-culture.md

```diff
@@ -8,7 +8,7 @@ We are a fast paced startup with really talented engineers who love open source
 
 Our values are reflected in the way we build software. 
 
-## At EOS Costa Rica we value:
+## At EOS Costa Rica We Value:
```

Comment:
At EOS Costa Rica we Value:

Reply:
no need to capitalize `we`

![image](https://user-images.githubusercontent.com/5632966/106681006-3f2e8080-6585-11eb-9d7e-b33183044e78.png)

Reply:
Solved